test/optics/optical_power_from_focus_distance_test.py

```diff
@@ -0,0 +1,31 @@
+from collections import namedtuple
+from pytest import fixture, raises
+from symplyphysics import (
+    assert_equal,
+    errors,
+    units,
+    Quantity, prefixes,
+)
+from symplyphysics.laws.optics import optical_power_from_focus_distance as optical_power_law
+
+#  Test example from https://bambookes.ru/stuff/reshenie_zadach/fizika/2-1-0-9505
+
+
+@fixture(name="test_args")
+def test_args_fixture():
+    f = Quantity(20 * prefixes.centi * units.meters)
+    Args = namedtuple("Args", ["f"])
+    return Args(f=f)
+
+
+def test_basic_power(test_args):
```

Comment:
We have updated type checking. Sync with master and add types, like this: https://github.com/blackyblack/symplyphysics/blob/master/test/chemistry/atomic_number_density_from_material_density_atomic_weight_test.py